.github/workflows/update-kernels.yml

```diff
@@ -2,6 +2,7 @@
 name: Update Kernels
 
 on:
+  workflow_dispatch:
```

Comment:
This is needed to trigger a refresh of kernels lists, allowing https://github.com/falcosecurity/dbg-go/pull/36 CI to be green.

Reply:
Anyway, we can leave it there, it makes no harm after all.